src/converter.js

```diff
@@ -21,9 +21,9 @@ converter._merge_property = (merge_type, property_name, destination_value, sourc
   if(source_value === undefined){
     return destination_value
   }
-	if(typeof destination_value === 'boolean' && typeof source_value === 'boolean'){
```

Comment:
Is it spaces instead of tabs for indentation, then? If so you should adjust the [`.editorconfig`](https://editorconfig.org/) file accordingly.

Reply:
When this file was rewritten by someone the previous tab code style was ignored :-( I'll update the editorconfig.